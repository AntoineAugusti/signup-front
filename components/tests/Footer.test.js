import ReactTestRenderer from 'react-test-renderer'

describe('components | Footer', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<Footer />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
