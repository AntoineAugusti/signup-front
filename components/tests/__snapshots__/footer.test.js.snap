// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Footer render should match snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="jsx-696190668 footer__logo"
    >
      <a
        className="jsx-696190668"
        href="https://api.gouv.fr/"
      >
        <img
          alt="Logo api.gouv.fr"
          className="jsx-696190668"
          src="/static/images/logo-api.gouv.fr-white.png"
        />
      </a>
      <ul
        className="jsx-696190668 footer__social"
      >
        <li
          className="jsx-696190668"
        >
          <a
            className="jsx-696190668"
            href="https://twitter.com/BetaGouv"
            title="Twitter"
          >
            <img
              alt="Twitter"
              className="jsx-696190668 icon icon-twitter"
              src="/static/images/social/twitter.svg"
            />
          </a>
        </li>
        <li
          className="jsx-696190668"
        >
          <a
            className="jsx-696190668"
            href="https://github.com/betagouv"
            title="Github"
          >
            <img
              alt="Github"
              className="jsx-696190668 icon icon-github"
              src="/static/images/social/github.svg"
            />
          </a>
        </li>
        <li
          className="jsx-696190668"
        >
          <a
            className="jsx-696190668"
            href="mailto:contact@particulier.api.gouv.fr"
            title="Nous Ã©crire un mail"
          >
            <img
              alt="Email"
              className="jsx-696190668 icon icon-mail"
              src="/static/images/social/email.svg"
            />
          </a>
        </li>
      </ul>
    </div>
    <ul
      className="footer__links"
    >
      <li>
        <h2>
          particulier.api.gouv.fr
        </h2>
      </li>
      <li>
        <a
          href="https://status.particulier.api.gouv.fr"
        >
          Status des API
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
